backend/app/core/config.py

```diff
@@ -3,11 +3,15 @@
 
 class Settings(BaseSettings):
     model_config = SettingsConfigDict(
-        env_file=".env", env_ignore_empty=True, extra="ignore"
+        env_file_encoding="utf-8",
+        env_file=".env",
+        env_ignore_empty=True,
+        extra="ignore",
     )
 
     CONNECTION_STRING: str
     DB_NAME: str
+    KAFKA_BROKERS: list[str]
 
 
 settings = Settings()
```

Comment:
from dotenv import load_dotenv

load_dotenv()
to load the env file

Reply:
You can read the [docs](https://docs.pydantic.dev/latest/concepts/pydantic_settings/#usage) this is how they do it.